pkgs/steamdeck-dsp/default.nix

```
@@ -44,12 +44,40 @@ stdenv.mkDerivation(finalAttrs: {
   postInstall = ''
     mv -vt $out $out/usr/*
     rmdir -v $out/usr
+
+    >&2 echo "Checking configuration files"
+
+    echo -e "${builtins.concatStringsSep "\n" finalAttrs.passthru.filesInstalledToEtc}" \
+      | sort >etc.expected.txt
+
+    for dir in wireplumber pipewire; do
+      find $out/share/$dir -type f -printf "$dir/%P\n" >>etc.unsorted.txt
+    done
+    sort <etc.unsorted.txt >etc.actual.txt
```

Couldn't this be done by redirecting the `for`, rather than have the indirection?

```suggestion
    for dir in wireplumber pipewire; do
      find $out/share/$dir -type f -printf "$dir/%P\n"
    done > etc.actual.txt
```

---

Good point, I was messing around with it incrementally.